### DISCUSSION TITLE
Hourly-scale rainfall runoff modeling

### DISCUSSION THREAD

**Discussion by mengziwen2000** [neuralhydrology/neuralhydrology#206] [Q&A] [answered]
Hi, based on ```Introduction.ipynb```, we can easily implement LSTM for daily scale rainfall runoff simulation. So, how to modify ```yml file``` to implement LSTM hourly scale rainfall runoff simulation. I checked out the Python library (https://neuralhydrology.readthedocs.io/en/latest/), which has some descriptions for modifying  yml file, for example:
```
use_frequencies.
  - 1H

train_start_date: “01/11/2021”
train_end_date: “31/03/2024”
test_start_date: “01/04/2024”
test_end_date: “11/11/2024”

predict_last_n. 
    1H: 1

predict_last_n: 1H: 1 
    1H: 365
```
But when I try it, it shows an error: ```ValueError: Unable to infer legitimate frequencies from the dataset: None.```

**Comment by gauchm** [comment, marked as answer]
That error typically means that your dataset doesn't have a consistent frequency, maybe because of missing time steps. You'll probably want to take a look at your input/target data and make sure that it is actually hourly.

**Reply by mengziwen2000** [reply]
Thank you for your reply. Yes, I checked the data and found that there is indeed a discontinuity in the data. Currently the hourly scale simulation is possible according to the configuration file I mentioned above
